styles.js

```diff
@@ -9,6 +9,7 @@ export default createGlobalStyle`
 
   body {
     margin: 0;
+    padding-top: 9.375rem;
```

Comment:
Maybe get a second opinion on this, i have never seen it done before like this, but i get what you are doing and i myself oftentimes just smash my head into my keyboard and hope my styling and positioning works, so i don't have an alternative solution, but this seems like it could mess with future responsiveness and positioning. I might be wrong about this so don't worry about this comment to much.

Reply:
Thank you for your comment, you're right. We will fix this later on in a seperate ticket for styling issues.